PetaPoco.Tests.Integration/Databases/BaseQueryLinqTests.cs

```diff
@@ -305,10 +308,8 @@ public virtual async void ExistsAsync_GivenPrimaryKeyMatchingNoRecord_ShouldBeFa
             (await DB.ExistsAsync<Person>(Guid.NewGuid())).ShouldBeFalse();
         }
 
-        /// <summary>
-        ///     Support the older syntax of starting with a WHERE clause.
-        /// </summary>
-        [Fact]
+        [Fact(DisplayName = "Support the older syntax of starting with a WHERE clause.")]
+        [Trait("Category", "Regression")]
         public virtual async void ExistsAsync_Regression_GivenSqlStringMatchingOneRecord_ShouldBeTrue()
         {
```

Comment:
This doesn't look right...shouldn't the regression tests *include* "WHERE" keyword, and the other tests exclude it?

Reply:
Found the related issue, makes sense now.